core/base/requirements-prod.txt

```diff
@@ -78,6 +78,7 @@ tabulate==0.9.0
 tenacity==8.2.3
 typing_extensions==4.11.0
 urllib3==2.2.1
+uvloop==0.21.0
```

Comment:
If you add it there you need to add it to requirements-dev.txt too

We didn't add it as we don't need great performance out of mta-sts-resolver

Reply:
Added